ovirt-web-ui.spec.in

```diff
@@ -20,19 +20,22 @@ Source0:        https://github.com/oVirt/ovirt-web-ui/archive/%{source_basename}
 
 ExclusiveArch:  x86_64
 
-BuildRequires: ovirt-engine-nodejs = 6.9.1
-# Keep ovirt-engine-nodejs-modules at particular version unless tested on higher
-BuildRequires: ovirt-engine-nodejs-modules = 0.0.17
+# Keep ovirt-engine-{nodejs|nodejs-modules|yarn} at particular version unless tested on higher
+# Please keep in sync with the automation/*.req files
+BuildRequires: ovirt-engine-nodejs=6.9.4
+BuildRequires: ovirt-engine-nodejs-modules=1.0.3
+BuildRequires: ovirt-engine-yarn=0.19.1
```

Comment:
I also think that to oVirt CI doesn't create `/etc/yum.repos.d/*.repo` files visible to the automation scripts, they are only visible to the mock tool. That means that the `yum-builddep` command will not work. I think you will have to remove it, or else ask request the CI team that those `.repo` files are created in the build root.